__tests__/index.spec.ts

```diff
@@ -1,17 +1,33 @@
 import * as path from 'path';
 import * as bodyParser from 'koa-bodyparser';
+import { ChowOptions } from 'oas3-chow-chow';
+import * as koa from 'koa';
 
 import { oas } from '../src';
 import { createContext } from './helpers/createContext';
-import { ChowOptions } from 'oas3-chow-chow';
 
-describe('Koa Oas3', () => {
-  const mw = oas({
-    file: path.resolve('./__tests__/fixtures/pet-store.json'),
-    endpoint: '/openapi',
-    uiEndpoint: '/openapi.html',
-    validatePaths: ['/pets']
-  });
+describe.only('Koa Oas3', () => {
```

Comment:
describe.only here, prob want to remove that change

Reply:
Done.